assets/js/googlesitekit/components-gm2/CircularProgress.stories.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import CircularProgress from './CircularProgress';
+
+const Template = () => (
```

Comment:
Hi, instead of having a big template with all the variations, can we have separate exports for all the variations. ie. `SecondaryCircularPtogress` like it is done on the `Button` stories and add a combined one for a single VRT scenario, same as the button stories.

Reply:
Updated, thank you!